frontend/src/components/create-user-page/CreateUser.scss

```diff
@@ -0,0 +1,44 @@
+@import "../../constants.scss";
+
+.form-heading {
```

Comment:
These class names are potentially a bit generic, could nest them to ensure they don't apply themselves to other forms? Or change the names themselves to be more specific? I dunno

Reply:
ok good point, i've made these more specific now